lib/Facebook.js

```diff
@@ -407,17 +409,18 @@ function Facebookbot(configuration) {
                 if (cb) { cb(null, facebook_botkit.webserver); }
             });
 
-
-        request.post('https://' + api_host + '/me/subscribed_apps?access_token=' + configuration.access_token,
-            function(err, res, body) {
-                if (err) {
-                    facebook_botkit.log('Could not subscribe to page messages');
-                } else {
-                    facebook_botkit.debug('Successfully subscribed to Facebook events:', body);
-                    facebook_botkit.startTicking();
-                }
-            });
-
+        var access_tokens = (typeof (configuration.access_token) === 'string') ? [configuration.access_token] : configuration.access_token;
```

Comment:
access_token can either be a string or an object. i used the [] notation to access the value by given the key as a variable, instead of dot notation, which works for known properties only.